### DISCUSSION TITLE
Erasing incorrect metadata

### DISCUSSION THREAD

**Discussion by gitST2020** [CodyBerenson/PGMA-Modernized#166] [General]
So I've been using PGMA for tagging but I'm finding that often, the tags are completed with incorrect data.  For example, the clip "(Timtales) - Franklin Saverio" didn't match for some reason and ended up labelling the Title as "Timtales Franklin Saverio".

How do I remove this metadata so that I can do a clean match search again?

I've tried manually editing it, but it's a laborious process.

Thanks

**Comment by JPH71** [comment]
To which agent are you matching it too...

If the video is in waybig for example, you would have to get its entry,
copy that and rename the video file as the entry.. put brackets around the
studio and take the year of the date of the entry, put brackets round that
and place it at the end of the string...

If you just have a title with just the name as you do.. plex may be getting
the data from the metadata tags of the video file...

for example for this entry
https://www.waybig.com/blog/2021/04/02/timtales-tim-kruger-re-fucks-creampies-saverio/
[image: image.png]
the video file would have to be named
(TimTales) - Tim Kruger Re-Fucks And Creampies Saverio (2021)

Then under the AdultGayScenes agent make sure that waybig is selected, and
local media assets are deselected.

Sorry if I come across a bit like a preacher...

Jason xxx

**Reply by gitST2020** [reply]
I have done what you suggest previously for a number of files but it's really time consuming and I was wondering if there's an easier way to get a match.

**Comment by JPH71** [comment]
The easiest way to clean out any metadata that plex collects is to move the
video file out of its current folder, then back in plex refresh the
library... this will remove it from plex, then put the video file back in
its original place and plex will re-scan it. if named as per one of the
agent websites and the agent is selected under Gay Adult Scenes... it
should bring down all the information...

Jason

**Reply by gitST2020** [reply]
Thanks for confirming that

**Comment by JPH71** [comment]
Then your best and quickest option is to refresh all metadata for the
library...

If the video files are named correctly.. they should be scraped
accordingly...

Or delete and recreate the library for a truly clean sweep

**Reply by jfreiman** [reply]
This latest suggestion, by manually refreshing the metadata is the best way --- as it allows viewing history to be maintained!  

If you remove the video from the folder, scan your library, and add the title back in your Library, then Plex creates a new view history for the "newly found" video in your library.

It would be great if, like the TV and Movie clients build into Plex, the PGMA Agent could retain the video's view history even if the video/file was moved to another folder (network share), or removed and then added back to the library as you first suggested.

**Comment by JPH71** [comment]
Have you tried unmatching it, then rematching...

It should maintain its add history that way...

**Comment by jfreiman** [comment]
Yea, unmatch, _clean bundles_, match works. If I don't do the clean bundles in between it sometimes leaves the old data and doesn't download/add the actor thumbnails.
